components/adblock_rust_ffi/src/lib.rs

```diff
@@ -497,8 +497,11 @@ pub unsafe extern "C" fn engine_hidden_class_id_selectors(
 
 #[cfg(feature = "ios")]
 #[no_mangle]
-pub unsafe extern "C" fn convert_rules_to_content_blocking(rules: *const c_char) -> *mut c_char {
+pub unsafe extern "C" fn convert_rules_to_content_blocking(rules: *const c_char, truncated: *mut bool) -> *mut c_char {
     use adblock::lists::{ParseOptions, RuleTypes};
+
+    const MAX_CB_LIST_SIZE: usize = 15000;
```

Comment:
Please comment where this number comes from so someone later can bump it if appropriate.

Reply:
Good idea, documented!